src/plugins/home/config.ts

```diff
@@ -33,6 +33,42 @@ import { schema, TypeOf } from '@osd/config-schema';
 export const configSchema = schema.object({
   disableWelcomeScreen: schema.boolean({ defaultValue: false }),
   disableNewThemeModal: schema.boolean({ defaultValue: false }),
+  prompts: schema.arrayOf(
+    schema.object({
+      prompt: schema.string(),
+      indexPattern: schema.string(),
+      datasourceName: schema.string(),
+      datasourceType: schema.string(),
+    }),
+    {
+      defaultValue: [
+        {
```

Comment:
should probably just make this completely optional.

Reply:
i think looking at it we are constructing the url anyways.

to make it less specific to having o11y. we can just have a prompt and url and just pass the url